src/Evolution/Systems/ScalarAdvection/VelocityField.hpp

```diff
@@ -0,0 +1,24 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
```

Comment:
you only need forward declarations for `DataVector` and `not_null`. You only need to include `Tensor/TypeAliases.hpp`